scripts/run_integration_tests_x86_64.sh

```diff
@@ -102,7 +102,7 @@ LINUX_CUSTOM_DIR="$WORKLOADS_DIR/linux-custom"
 if [ ! -f "$VMLINUX_IMAGE" ]; then
     SRCDIR=$PWD
     pushd $WORKLOADS_DIR
-    time git clone --depth 1 "https://github.com/cloud-hypervisor/linux.git" -b "ch-5.12" $LINUX_CUSTOM_DIR
+    time git clone --depth 1 "https://github.com/cloud-hypervisor/linux.git" -b "ch-5.13" $LINUX_CUSTOM_DIR
```

Comment:
I would prefer to embed the full version number (e.g. 5.13rc5)

Reply:
Oh yes I can do that!